src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/repository/PrisonerIepLevelRepository.kt

```diff
@@ -10,6 +10,5 @@ interface PrisonerIepLevelRepository : CoroutineCrudRepository<PrisonerIepLevel,
   fun findAllByPrisonerNumberOrderByReviewTimeDesc(prisonerNumber: String): Flow<PrisonerIepLevel>
   fun findAllByBookingIdOrderByReviewTimeDesc(bookingId: Long): Flow<PrisonerIepLevel>
   fun findAllByBookingIdInAndCurrentIsTrueOrderByReviewTimeDesc(bookingIds: List<Long>): Flow<PrisonerIepLevel>
-  suspend fun findFirstByBookingIdAndCurrentIsTrueOrderByReviewTimeDesc(bookingId: Long): PrisonerIepLevel?
```

Comment:
Thought about refactoring this one to drop `First` so it returns `Flow<PrisonerIepLevel>` but decided to remove it and instead use `findAllByBookingIdInAndCurrentIsTrueOrderByReviewTimeDesc` so the repo has fewer methods. Happy to discuss if folks would prefer a different approach

Reply:
It's fine, technically in this case you don't need to sort by review time but that's fine :)